### PR TITLE
Add some basic CI

### PR DESCRIPTION
**Pull request by calebschoepp**
## Changes
- Run unit tests in CI
- Run linting in CI

## Notes
- Need to un-comment rest of CI once we start using Firebase

### CODE REVIEW

File: .github/test.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+      # Step 3: Check the code with Android linter
+      - name: Run Android Linter
+        run: ./gradlew lintStagingDebug
+
+      # Step 4: Yun your unit tests
+      - name: Run Unit Tests
+        run: ./gradlew testStagingDebugUnitTest
+
+      # Step 4: Assemble debug apk to send to firebase test lab
```
Comment (cyrusdiego) [inline .github/test.yml]: So I just wanted to confirm these steps: it seems these steps are for instrumentation test? And then we would assembly our apk and send to firebase test labs. Is my understanding correct?

**Review by cyrusdiego — CHANGES_REQUESTED** [review]

File: .github/test.yml (hunk 1 above)
Comment (calebschoepp) [reply .github/test.yml]: Yep that's right. See here for more details. https://www.coletiv.com/blog/android-github-actions-setup/

File: .github/test.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      # - name: Decode google-services.json
+      #   env:
+      #     FIREBASE_CONFIG: ${{ secrets.FIREBASE_CONFIG }}
+      #   run: echo $FIREBASE_CONFIG > app/google-services.json
+
+      # Step 3: Check the code with Android linter
+      - name: Run Android Linter
+        run: ./gradlew lintStagingDebug
+
+      # Step 4: Yun your unit tests
```
[the remark this answers is not in the document]
Comment (calebschoepp) [reply .github/test.yml]: Nice

**Comment by calebschoepp** [comment]
FYI: I know tests are failing, just need to work on something else this afternoon. Will fix this later tonight hopefully

**Review by cyrusdiego — CHANGES_REQUESTED** [review]
The build seems to be failing still, I think we need to have an assembleStagingDebug file in our project directory?

**Comment by calebschoepp** [comment]
@cyrusdiego the tests are passing now. Just waiting for a LGTM!

**Review by cyrusdiego — APPROVED** [review]
LGTM :D

**Pull request merged**